src/Components/CardList.tsx

```diff
@@ -316,7 +316,7 @@ export const CardList = ({
   if (filteredServices === null) return <></>
 
   return filteredServices && filteredServices.length > 0 ? (
-    <Content>
```

Comment:
Some of the pages that contain a CardList already have a main, so we will end up with two